modules/openapi-generator/src/main/resources/csharp/libraries/generichost/SourceGenerationContext.mustache

```diff
@@ -1,9 +1,8 @@
 {{#useSourceGeneration}}
-
     /// <summary>
     /// The {{classname}}SerializationContext
     /// </summary>
     [JsonSourceGenerationOptions(WriteIndented = true, GenerationMode = JsonSourceGenerationMode.Metadata | JsonSourceGenerationMode.Serialization)]
     [JsonSerializable(typeof({{classname}}))]
     {{>visibility}} partial class {{classname}}SerializationContext : JsonSerializerContext { }
-{{/useSourceGeneration}}
\ No newline at end of file
+{{/useSourceGeneration}}
```

Comment:
Please remove the spacing changes in this file.

Reply:
Done.